controllers/memberoperatorconfig/configuration.go

```diff
@@ -28,6 +145,52 @@ func (a MemberStatusConfig) RefreshPeriod() time.Duration {
 	return d
 }
 
+type ToolchainClusterConfig struct {
+	t toolchainv1alpha1.ToolchainClusterConfig
+}
+
+func (a ToolchainClusterConfig) HealthCheckPeriod() time.Duration {
+	defaultClusterHealthCheckPeriod := "10s"
+	healthCheckPeriod := getString(a.t.HealthCheckPeriod, defaultClusterHealthCheckPeriod)
+	d, err := time.ParseDuration(healthCheckPeriod)
+	if err != nil {
+		d, _ = time.ParseDuration(defaultClusterHealthCheckPeriod)
+	}
+	return d
+}
+
+func (a ToolchainClusterConfig) HealthCheckTimeout() time.Duration {
+	defaultClusterHealthCheckTimeout := "3s"
+	healthCheckTimeout := getString(a.t.HealthCheckTimeout, defaultClusterHealthCheckTimeout)
+	d, err := time.ParseDuration(healthCheckTimeout)
+	if err != nil {
+		d, _ = time.ParseDuration(defaultClusterHealthCheckTimeout)
+	}
+	return d
+}
+
+type WebhookConfig struct {
+	w toolchainv1alpha1.WebhookConfig
+}
+
+func (a WebhookConfig) Deploy() bool {
+	return getBool(a.w.Deploy, true)
+}
+
+func getBool(value *bool, defaultValue bool) bool {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
+
+func getInt(value *int, defaultValue int) int {
+	if value != nil {
+		return *value
+	}
+	return defaultValue
+}
```

Comment:
It's not critical, but it could be moved to toolchain-common so it can be reused in other repos

Reply:
Put up a PR for this: https://github.com/codeready-toolchain/toolchain-common/pull/196